testing/tests/DevExpress.ui.widgets/overlay.tests.js

```diff
@@ -473,6 +473,35 @@ testModule('option', moduleConfig, () => {
         assert.strictEqual(onResizeFired.callCount, 1, 'onResize fired');
         assert.strictEqual(onResizeEndFired.callCount, 1, 'onResizeEnd fired');
     });
+
+    test('wrapperAttr defaultOption', function(assert) {
```

Comment:
we should not test `defaultOptions` method for every new option. Let's just add new config to `defaultOptions.test.js`